geometry_msgs/Vector3 linear_input
geometry_msgs/Vector3 angular_input
float32 controller_yaw
float32 linear_scale
string command_type
string controlled_id